content/enterprise_influxdb/v1.10/administration/monitor/monitor-with-cloud.md

```diff
@@ -178,8 +178,8 @@ Send a notification to PagerDuty or HTTP endpoints (other webhooks) by [upgradin
 1. Go to **Alerts > Notification Rules** and click **{{< icon "plus" "v2" >}} Create**. 
 2. Fill out the **About** and **Conditions** section then click **Create Notification Rule**. 
 
-## Monitor with InfluxDB Insights and Aware 
+## Monitor with InfluxDB Insights
 
-For InfluxDB Enterprise customers, Insights and Aware are free services that can monitor your data. InfluxDB Insights sends your data to a private Cloud account and will be monitored with the help of the support team. InfluxDB Aware is a similar service, but you monitor your data yourself. 
+For InfluxDB Enterprise customers, Insights is a free services that can monitor your cluster. InfluxDB Insights sends monitoring metrics for your cluster to a private Cloud account. This allows the support team to monitor your cluster health as well as making resource usage statistics available to assist with support tickets that you raise.
 
 To apply for this service, please contact the [InfluxData Support team](mailto:support@influxdata.com). 
```

Comment:
@jstirnaman I've deliberately not adjusted the link here, because I didn't want this PR to cause a merge conflict with your PR https://github.com/influxdata/docs-v2/pull/5003